pkg/osquery/tables/wifi_networks/testdata/output.txt

```diff
@@ -0,0 +1,18 @@
+dot11Ssid             : NativeWifi.Wlan+Dot11Ssid
+phyId                 : 2
+dot11Bssid            : {128, 42, ..., 235...}
+dot11BssType          : Infrastructure
+dot11BssPhyType       : 7
+rssi                  : -45
+linkQuality           : 100
+inRegDomain           : True
+beaconPeriod          : 0
+timestamp             : 610829842048
+hostTimestamp         : 132550441422028881
```

Comment:
These timestamps don't look like unix time. Should we do anything to convert them, or simply present data as-is?

Reply:
tbh I don't know anything about them. I think they're documented by MS, but they weren't part of the original dataset I was collecting, so I haven't looked into it. I can take a peek and at least document it here.

Reply:
We can kick that forward. I think osquery knows how to parse them, so we might have some example c code